.github/workflows/ci-rootless.yml

```diff
@@ -47,6 +47,14 @@ jobs:
       checks: write
     steps:
       - uses: actions/checkout@v4
+      - name: Install Latest Docker
+        run: |
+          sudo systemctl stop docker.service
+          sudo apt-get purge docker-ce docker-ce-cli containerd.io docker-compose-plugin docker-ce-rootless-extras docker-buildx-plugin
+          curl -fsSL https://test.docker.com -o get-docker.sh
+          sudo REPO_FILE=docker-ce-staging.repo DOWNLOAD_URL=https://download-stage.docker.com sh ./get-docker.sh
```

Comment:
These currently use the staging domain (packages will be deployed to production after the weekend)